src/components/SearchForm/SearchFormOptions.tsx

```diff
@@ -0,0 +1,132 @@
+import COLON_MEDICATION_CODES from '../../queries/mockData/colonMedicationCodes.json';
+import COLON_RADIATION_CODES from '../../queries/mockData/colonRadiationCodes.json';
+import COLON_SURGERY_CODES from '../../queries/mockData/colonSurgeryCodes.json';
+import {
+  default as BRAIN_CANCER_SUBTYPE_CODES,
+  default as BRAIN_RADIATION_CODES,
+  default as BRAIN_SURGERY_CODES,
+} from '../../queries/mockData/empty.json';
+import LUNG_CANCER_SUBTYPE_CODES from '../../queries/mockData/lungCancerSubTypeCodes.json';
+import LUNG_CANCER_TYPE_CODES from '../../queries/mockData/lungCancerTypeCodes.json';
+import LUNG_MEDICATION_CODES from '../../queries/mockData/lungMedicationCodes.json';
+import LUNG_RADIATION_CODES from '../../queries/mockData/lungRadiationCodes.json';
+import LUNG_SURGERY_CODES from '../../queries/mockData/lungSurgeryCodes.json';
+import MULTIPLE_MYELOMA_CANCERTYPE_CODES from '../../queries/mockData/MultipleMyelomaCancerTypeCodes.json';
+import MULTIPLE_MYELOMA_MEDICATION_CODES from '../../queries/mockData/MultipleMyelomaMedicationCodes.json';
+import PROSTATE_CANCER_TYPE_CODES from '../../queries/mockData/prostateCancerTypeCodes.json';
+
+export type CancerTypeDetails = {
+  category: string;
+  cancerCodes: CodedValueType[];
+  cancerSubtype: CodedValueType[];
+  biomarkers: CodedValueType[];
+  surgeryCodes: CodedValueType[];
+  medications: CodedValueType[];
+  stages: CodedValueType[];
+  radiationCodes: CodedValueType[];
+};
+
+export const cancerTypeDetails: Record<string, CancerTypeDetails> = {
+  lung: buildCancerCodeJSON(
+    'lung',
+    LUNG_CANCER_TYPE_CODES.entry,
+    LUNG_CANCER_SUBTYPE_CODES.entry,
+    LUNG_MEDICATION_CODES.entry,
+    LUNG_RADIATION_CODES.entry,
+    LUNG_SURGERY_CODES.entry,
+    BREAST_CANCER_STAGES.entry,
+    BREAST_CANCER_BIOMARKERS.entry
+  ),
+  colon: buildCancerCodeJSON(
+    'colon',
+    COLON_CANCER_TYPE_CODES.entry,
+    null,
+    COLON_MEDICATION_CODES.entry,
+    COLON_RADIATION_CODES.entry,
+    COLON_SURGERY_CODES.entry,
+    BREAST_CANCER_STAGES.entry,
+    null
+  ),
+  brain: buildCancerCodeJSON(
+    'brain',
+    BRAIN_CANCER_TYPE_CODES.entry,
+    BRAIN_CANCER_SUBTYPE_CODES.entry,
+    BRAIN_MEDICATION_CODES.entry,
+    BRAIN_RADIATION_CODES.entry,
+    BRAIN_SURGERY_CODES.entry,
+    BREAST_CANCER_STAGES.entry,
+    BRAIN_BIOMARKER_CODES.entry
+  ),
+  prostate: buildCancerCodeJSON(
+    'prostate',
+    PROSTATE_CANCER_TYPE_CODES.entry,
+    null,
+    null,
+    null,
+    null,
+    BREAST_CANCER_STAGES.entry,
+    null
+  ),
+  breast: buildCancerCodeJSON(
+    'breast',
+    BREAST_CANCER_TYPE_CODES.entry,
+    null,
+    BREAST_CANCER_MEDICATIONS.entry,
+    null,
+    null,
+    BREAST_CANCER_STAGES.entry,
+    BREAST_CANCER_BIOMARKERS.entry
+  ),
+  mm: buildCancerCodeJSON(
+    'mm',
+    MULTIPLE_MYELOMA_CANCERTYPE_CODES.entry,
+    null,
+    MULTIPLE_MYELOMA_MEDICATION_CODES.entry,
+    null,
+    null,
+    BREAST_CANCER_STAGES.entry,
+    null
+  ),
+};
```

Comment:
Either the nulled values should have comments explaining why they aren't needed, or the function should be refactored to take an options object that only contains the required keys.

Reply:
replaced nulls with empty arrays.  Added comments where appropriate